cmd/server/assets/realmkeys.html

```diff
@@ -0,0 +1,262 @@
+            </div>
+          </div>
+          <div class="form-group row">
+            <label for="name" class="col-sm-3">Public key:</label>
+            <div class="col-sm-9">
+            {{if .certPublicKey}}
+            <div class="alert alert-secondary" role="alert">
+            <pre>{{.certPublicKey}}</pre>
+            </div>
+            {{else}}
+            <div class="alert alert-danger" role="alert">{{.certPublicKeyError}}</div>
+            {{end}}
+            </div>
+          </div>
+        </div>
+      </div>
+    {{end}}
+      
+    <div class="card mb-3">
+        <div class="card-header">
+          Realm signing key configuration
+        </div>
+        <div class="card-body">
+        {{if .supportsPerRealmSigning}}            
+          {{if not $realm.UseRealmCertificateKey}}
+            {{if $realm.CanUpgradeToRealmSigningKeys}}
+            <form method="POST" action="/realm/keys/upgrade">
+                {{ .csrfField }}
+                <div class="form-group row">
+                  <div class="col-sm-12">
+                    <div class="alert alert-warning" role="alert">
+                        <strong>Unerversable action:</strong> Before upgrading to realm specific signing keys, make sure that you have
+                        configured the signing key information on your exposure notifications key server.
+                        Once you upgrade to realm spcific signing keys, this realm cannot be changed
+                        back to using the system signer. This change will be effective immediatly.
+                    </div>
+                    <button type="submit" class="btn btn-primary btn-block">Upgrade to realm specific signing keys</button>
+                  </div>
+                </div>
+            </form>
+            {{else}}
+            <div class="alert alert-info" role="alert">
+                After setting your realm's issuer and audience settings, you can enable
+                signing verification certificates with realm specific keys.
+            </div>
+            {{end}}
+          {{end}}
+
+          {{if $realm.UseRealmCertificateKey}}
+          <div class="alert alert-info" role="alert">
+            This information needs to be communicated to the operator
+            of you exposure notification key server in order to be able to utilize
+            verification certificates issued by this server.
+          </div>
+          {{end}}
+
+          <form method="POST" action="/realm/keys/save">
+            {{ .csrfField }}
+            <div class="form-group row">
+                <label for="name" class="col-sm-3">Issuer (iss):</label>
+                <div class="col-sm-9">
+                  <input type="text" id="iss" name="iss" class="form-control" value="{{$realm.CertificateIssuer}}" />
+                  <small class="form-text text-muted">
+                    Once this value is configured on the key server, it should not be changed.
```

Comment:
Can we provide different help text here based on whether the value exists or is empty? It seems like we want to provide slightly different hints depending on whether this is the first time or not.